src/component/navigation/Navigation.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import  {Link}  from "react-router-dom";
+
+import { style } from "./style";
+
+const Navigation = () => {
+  const [hoveredProduct, setHoveredProduct] = useState(null);
+
+  const handleMouseEnter = (product) => {
+    setHoveredProduct(product);
+  };
+
+  const handleMouseLeave = () => {
+    setHoveredProduct(null);
+  };
+
+  const isProductHovered = (product) => {
+    return hoveredProduct === product;
+  };
+
+  return (
+    <Box
+      style={style.navigationmainBox}
+    >
+      <ProductBox
+        product="Mart"
+        imageSrc="martimg.png"
+        isHovered={isProductHovered("Mart")}
+        onMouseEnter={() => handleMouseEnter("Mart")}
+        onMouseLeave={handleMouseLeave}
+      />
+
+      <ProductBox
+        product="Beauty"
+        imageSrc="beautyimg.png"
+        isHovered={isProductHovered("Beauty")}
+        onMouseEnter={() => handleMouseEnter("Beauty")}
+        onMouseLeave={handleMouseLeave}
+      />
+
+      <ProductBox
+        product="Home & Decor"
```

Comment:
why using repeatitions.
always use html one time, 
you can make the data as array and map it to return component once only